### PR TITLE
product view dyanmic crop

### PR DESCRIPTION
**Pull request by stephaniepark**

### CODE REVIEW

**Comment by paulchang333** [comment]
LGTM!

File: FarmBuddy/app/src/main/java/com/example/farmbuddy/ui/productview/ProductViewFragment.kt [hunk 1]
```diff
@@ -34,7 +34,7 @@ class ProductViewFragment : Fragment() {
     private var _binding: FragmentProductViewBinding? = null
 
     private val binding get() = _binding!!
-    private var prodID = " "
+    var prodID = " "
```
Comment (stephaniepark) [inline FarmBuddy/app/src/main/java/com/example/farmbuddy/ui/productview/ProductViewFragment.kt:37]: @JacobRintjema

File: FarmBuddy/app/src/main/java/com/example/farmbuddy/ui/productview/ProductViewFragment.kt [hunk 2]
```diff
@@ -48,11 +48,12 @@ class ProductViewFragment : Fragment() {
         val productViewModel =
             ViewModelProvider(this).get(ProductViewModel::class.java)
 
-        prodID = "1ab1f3a6-231c-46c9-a9c5-48e73c0aa6f0"
+        prodID = " "
```
Comment (stephaniepark) [inline FarmBuddy/app/src/main/java/com/example/farmbuddy/ui/productview/ProductViewFragment.kt:51]: @JacobRintjema

File: FarmBuddy/app/src/main/java/com/example/farmbuddy/ui/productview/ProductViewFragment.kt [hunk 3]
```diff
@@ -62,17 +63,22 @@ class ProductViewFragment : Fragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
```
Comment (stephaniepark) [inline FarmBuddy/app/src/main/java/com/example/farmbuddy/ui/productview/ProductViewFragment.kt:63]: all changes here

**Pull request merged**